index.html

```diff
@@ -0,0 +1,135 @@
+      <ul class="menu">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#main-program-home">Program</a></li>
+        <li><a href="#btn-main-program-home">Join</a></li>
+        <li><a href="#partners-crypto-con">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#" class="cc-campaign">CC Crypto</a></li>
+      </ul>
+    </nav>
+
+    <header class="header-home">
+      <p class="home-greeting">"Hello Crypto Lovers!"</p>
+      <h1 class="title-header-home">
+        CRYPTO-CON <br />
+        <span>2022</span>
+      </h1>
+      <p class="text-header-home">
+        The one and only <strong>CRYPTO-CON</strong> is where you will find the
+        most relevant with cryptocurrencies. You will be able to see projects,
+        meet other Crypto Lovers from over 80 countries, and much more!
+      </p>
+      <p class="date-header-home">
+        2022.03.04(FRI) ~ 6(SUN) <br />
+        <span>
+          @ Centro de Convenciones y Exposiciones of Aguascalientes, Mexico.
+        </span>
+      </p>
+    </header>
+    <section class="main-program-home" id="main-program-home">
+      <h2 class="title-main-program-home">Main Program</h2>
+      <hr />
+      <ul class="programs-main-program-home">
+        <li>
+          <img src="resources/icons/presentation.svg" alt="">
+          <h3>Lecture</h3>
+          <p>
+            Listen to the speakers from various countries about the messeges of
+            sharing and opening
+          </p>
+        </li>
+        <li>
+            <img src="resources/icons/lecture.svg" alt="">
+          <h3>Exhibition</h3>
+          <p>Appreciate various creations applying CC license of artists.</p>
+        </li>
+        <li>
+            <img src="resources/icons/chat.svg" alt="">
+          <h3>Forum</h3>
+          <p>
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </li>
+        <li>
+            <img src="resources/icons/puzzle.svg" alt="">
+          <h3>Workshop</h3>
+          <p>
+            Try creating your own work using open source license rather that
+            just watching.
+          </p>
+        </li>
+        <li>
+            <img src="resources/icons/network.svg" alt="">
+          <h3>Ignite</h3>
+          <p>
+            Get opportunities to network with CC affiliates around the world,
+            also after the Con.
+          </p>
+        </li>
+      </ul>
+      <button type="button" class="btn-main-program-home" id="btn-main-program-home">
+        JOIN CRYPTO-CON 2022
+      </button>
+      <a class="link-main-program-home" href="#" id="btn-main-program-home1"
+        >SEE THE WHOLE PROGRAMS</a
+      >
+    </section>
+    <section class="featured-speakers">
+      <h2 class="title-featured-speakers">Featured Speakers</h2>
+      <hr />
+      <ul class="speakers-list"></ul>
+    </section>
```

Comment:
- [ ]  I think we can improve this design by using the exact same colors, for text, backgrounds, borders, etc, from the original design. Basic color swatch: #272A31, #EC5242, #D3D3D3

* Basic color swatch: #272A31, #EC5242, #D3D3D3

your design:

![p1](https://user-images.githubusercontent.com/57562869/152656953-98edfb1b-20aa-4f63-87e8-d03ec41f97f0.png)


figma design:

![p2](https://user-images.githubusercontent.com/57562869/152657038-7e6e76e4-8847-4180-9eee-f927ba1bc45d.png)